install-dev

```diff
@@ -0,0 +1,53 @@
+: ${GITHUB_USERNAME:=hortonworks}
+: ${GITHUB_PROJECT:=cloudbreak-deployer}
+ 
+circleciapi="https://circleci.com/api/v1.1"
+circle() {
+    declare path=$1
+    shift
+    
+    local url="$circleciapi/$path"
+    curl -s -G \
+        -H "Accept: application/json" \
+        "$url" "$@"
+}
+
+main() {
+    declare branch=${1:-master}
+
+    local os=$(uname -s | tr "[A-Z]" "[a-z]")
+    local latestartifacts="project/github/$GITHUB_USERNAME/$GITHUB_PROJECT/latest/artifacts?branch=$branch&filter=successful"
+    local artifacts=$(circle $latestartifacts | tr -d '[:space:]')
+    local message=$(echo $artifacts | jq -c ".message" 2>/dev/null)
```

Comment:
`jq` is a third party tool, most of the operating systems doesn't contain by default. .deps folder contains `jq` under bin, but requires to initialize `cbd` first.